### PR TITLE
Moved categories from stories into guides

### LINKED ISSUE #341: [Bug]: Fix CMS category configuration
## Description

In the CMS the category select configuration is on stories, not guides

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Fixes #341 

## Description

Fixes CMS missconfiguration

@geeksforsocialchange/developers

### CODE REVIEW

File: public/admin/config.yml [hunk 1]
```diff
@@ -99,6 +85,20 @@ collections:
...
+          name: "Category",
+          widget: "select",
+          options:
+            [
+              "admin-and-info",
+              "media-publicity-events",
+              "working-with-people",
+              "working-with-the-authorities",
+            ],
+          default: "admin-and-info",
```
Comment (katjam) [inline public/admin/config.yml:99]: Probably don't want a default. Would be too easy to add to wrong category by accident.

File: public/admin/config.yml [hunk 2]
```diff
@@ -99,6 +85,20 @@ collections:
           pattern: ["^.{50,160}$", "Must be between 50 and 160 characters"],
           i18n: true,
         }
+      - {
+          label: "Category",
+          name: "Category",
+          widget: "select",
+          options:
+            [
+              "admin-and-info",
```
Comment (katjam) [inline public/admin/config.yml:94]: Could use un hyphonated strings as per given so content people don't have to think as much but if this is all wired up already, keep!

File: public/admin/config.yml [hunk 3]
```diff
@@ -99,6 +85,20 @@ collections:
           pattern: ["^.{50,160}$", "Must be between 50 and 160 characters"],
           i18n: true,
         }
+      - {
+          label: "Category",
```
Comment (katjam) [inline public/admin/config.yml:89]: Did you also assign these to the existing guides in markdown?

Comment (ivan-kocienski-gfsc) [reply public/admin/config.yml:95]: yes, for the stories that are in the repo

File: public/admin/config.yml (hunk 2 above)
Comment (ivan-kocienski-gfsc) [reply public/admin/config.yml:100]: Actually it is possible to configure the select to have Key:Value on these fields

Comment (ivan-kocienski-gfsc) [reply public/admin/config.yml]: done

File: public/admin/config.yml (hunk 1 above)
Comment (ivan-kocienski-gfsc) [reply public/admin/config.yml]: done

File: public/admin/config.yml [hunk 4]
```diff
@@ -59,7 +45,13 @@ collections:
         required: false
         i18n: true
         fields:
-          - { label: "Image", name: "src", widget: "image", i18n: true }
+          - {
+              label: "Image",
+              name: "src",
+              widget: "image",
+              i18n: true,
+              hint: "Please keep file size under 1 megabyte.",
```
Comment (katjam) [inline public/admin/config.yml:53]: This is in the wrong branch?

File: public/admin/config.yml [hunk 5]
```diff
@@ -99,6 +91,20 @@ collections:
           pattern: ["^.{50,160}$", "Must be between 50 and 160 characters"],
           i18n: true,
         }
+      - {
+          label: "Category",
+          name: "Category",
+          widget: "select",
+          options:
+            [
+              { label: "Community Groups - Admin and Information", value: "admin-and-info" },
```
Comment (katjam) [inline public/admin/config.yml:100]: perfect!

**Review by katjam — CHANGES_REQUESTED** [review]
You need to back out the image changes from this one, but the CMS change looks good.

**Review by aaaaargZombies — APPROVED** [review]
nice, glad you caught this

**Pull request merged**